<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="4" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-21T13:43:22 EET" name="Default suite" finished-at="2024-09-21T13:43:35 EET" duration-ms="12921">
    <groups>
    </groups>
    <test started-at="2024-09-21T13:43:22 EET" name="Default test" finished-at="2024-09-21T13:43:35 EET" duration-ms="12921">
      <class name="myTestCases">
        <test-method is-config="true" signature="mysetup()[pri:0, instance:myTestCases@1d71006f]" started-at="2024-09-21T13:43:22 EET" name="mysetup" finished-at="2024-09-21T13:43:22 EET" duration-ms="308" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mysetup -->
        <test-method signature="AdddataToTheDataBase()[pri:1, instance:myTestCases@1d71006f]" started-at="2024-09-21T13:43:22 EET" name="AdddataToTheDataBase" finished-at="2024-09-21T13:43:22 EET" duration-ms="385" status="FAIL">
          <exception class="java.sql.SQLIntegrityConstraintViolationException">
            <message>
              <![CDATA[Duplicate entry '504' for key 'customers.PRIMARY']]>
            </message>
            <full-stacktrace>
              <![CDATA[java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '504' for key 'customers.PRIMARY'
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:109)
at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
at com.mysql.cj.jdbc.StatementImpl.executeUpdateInternal(StatementImpl.java:1537)
at com.mysql.cj.jdbc.StatementImpl.executeLargeUpdate(StatementImpl.java:2384)
at com.mysql.cj.jdbc.StatementImpl.executeUpdate(StatementImpl.java:1410)
at myTestCases.AdddataToTheDataBase(myTestCases.java:62)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.sql.SQLIntegrityConstraintViolationException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AdddataToTheDataBase -->
        <test-method signature="updateDataBase()[pri:2, instance:myTestCases@1d71006f]" started-at="2024-09-21T13:43:22 EET" name="updateDataBase" finished-at="2024-09-21T13:43:22 EET" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateDataBase -->
        <test-method signature="Getdata()[pri:3, instance:myTestCases@1d71006f]" started-at="2024-09-21T13:43:22 EET" name="Getdata" finished-at="2024-09-21T13:43:35 EET" duration-ms="12191" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Getdata -->
      </class> <!-- myTestCases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
